test/functoria/e2e/help.t

```diff
@@ -68,23 +68,19 @@ Test that the help command works without config file:
              The topic to get help on.
   
   OPTIONS
-         --help[=FMT] (default=auto)
-             Show this help in format FMT. The value FMT must be one of `auto',
-             `pager', `groff' or `plain'. With `auto', the format is `pager` or
-             `plain' whenever the TERM env var is `dumb' or undefined.
-  
          --man-format=FMT (absent=pager)
-             Show output in format FMT. The value FMT must be one of `auto',
-             `pager', `groff' or `plain'. With `auto', the format is `pager` or
-             `plain' whenever the TERM env var is `dumb' or undefined.
-  
-         --version
-             Show version information.
+             Show output in format FMT. The value FMT must be one of auto,
+             pager, groff or plain. With auto, the format is pager or plain
+             whenever the TERM env var is dumb or undefined.
   
   COMMON OPTIONS
          --color=WHEN (absent=auto)
-             Colorize the output. WHEN must be one of `auto', `always' or
-             `never'.
+             Colorize the output. WHEN must be one of auto, always or never.
+  
```

Comment:
@dbuenzli and that line also contains a whitespace (instead of being empty)

Reply:
I don't get it. The space here is to seperate it from the next option. The space already existed before on line 84.

Reply:
Ah you mean there's space on the line !? Yeah maybe, sorry about that…

Reply:
yes, there are two `' '` characters on that line. It's really really minor though :-)

Reply:
Are you sure ? github's interface is not showing it, and I have no idea were it could come from as it happens only here no ?

Reply:
It seems that the _upstream_ version of `cmdliner` fix/delete these whitespace.